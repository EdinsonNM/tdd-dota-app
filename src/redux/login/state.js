export default {
    loading: false,
    error: false,
    player: null,
};
