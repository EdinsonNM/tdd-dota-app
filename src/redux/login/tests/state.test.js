import state from "../state";
describe("state", () => {
    test("should return initial state", () => {
        expect(state).toMatchSnapshot();
    });
});
