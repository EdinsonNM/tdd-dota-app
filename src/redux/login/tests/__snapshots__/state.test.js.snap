// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`state should return initial state 1`] = `
Object {
  "error": false,
  "loading": false,
  "player": null,
}
`;
